docs/conf.py

```diff
@@ -92,9 +92,10 @@ def __getattr__(cls, name):
 # built documents.
 #
 # The short X.Y version.
-version = '0.0'
+import fuel
+version = fuel.__version__[:fuel.__version__.rindex('.')]
```

Comment:
Again, tiny nitpick, but if you don't want to assume that there are no more than 3 dots:
```
v[::-1].split('.', 1)[-1][::-1]
```
or
```
re.match(r'(.*?\..*?)\.', v).expand(r'\1')
```